extensions/README.md

```diff
@@ -0,0 +1,37 @@
+# NEAR CLI extensions system
+`NEAR CLI` is built to scale. The number of possible features is endless. Instead of choosing only some of them, we are creating an `Extensions System` that will empower our users to choose, build and share `NEAR CLI` functionality.
+
+## How it works
+Extensibility is achieved by translating a `NEAR CLI` invocation of the form `near (?<command>[^ ]+)` into an invocation of an external tool `near-${command}` that then needs to be present in one of the user's `$PATH` directories.
+It means that you can write it in any language and with the use of any framework, it just needs to be called `near-cli-{extension-name}` and be installed on your system. This approach is inspired by [Cargo](https://github.com/rust-lang/cargo).
```

Comment:
```suggestion
It means that you can write it in any language and with the use of any framework, it just needs to be called `near-{extension-name}` and be installed on your system. This approach is inspired by [Cargo](https://github.com/rust-lang/cargo).
```

Reply:
It's `near-cli` intensionally. `near-cli` part is dropped in code. `near-{}` is too generic.

Reply:
@volovyk-s If that is the case, then the line above is incorrect. Actually, I believe if `near` is not too generic then `near-{}` is also not too generic. I vote for consistency, so if the main binary is named as `near` then the extensions should also be `near-{}` and if the main binary is `near-cli` then the extensions should have `near-cli-{}` naming (should we consider renaming it to `near`?), otherwise I feel it will be a major source of questions "why near does not see my extension?"